openml/datasets/functions.py

```diff
@@ -405,7 +412,9 @@ def get_dataset(
         features_file = _get_dataset_features_file(did_cache_dir, dataset_id)
 
         try:
-            qualities_file = _get_dataset_qualities_file(did_cache_dir, dataset_id)
+            qualities_file = _get_dataset_qualities_file(
```

Comment:
I think it makes more sense to use `download_qualities` here and not call the `_get_dataset_qualities_file` if we don't want to process the qualities.